src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/ApiController.cs

```diff
@@ -128,6 +140,113 @@ public async Task<IActionResult> Create(CreateApiViewModel model)
             return BadRequest(ModelState);
         }
 
+        [HttpPost]
+        [Route("disable/{tenantName}")]
+        public async Task<IActionResult> Disable(string tenantName)
+        {
+            if (!_currentShellSettings.IsDefaultShell())
+            {
+                return Forbid();
+            }
+
+            if (!await _authorizationService.AuthorizeAsync(User, Permissions.ManageTenants))
+            {
+                return this.ChallengeOrForbid("Api");
+            }
+
+            if (!_shellHost.TryGetSettings(tenantName, out var shellSettings))
+            {
+                return NotFound();
+            }
+
+            if (shellSettings.State != TenantState.Running)
+            {
+                return Problem(
```

Comment:
Okay will change it.

Yes because it has an additional `detail` parameter (that produces a `ProblemDetails` response) that I planned to use, like line 238.